Assets/Scripts/Network/RoomCreator.cs

```diff
@@ -21,22 +23,30 @@ public void CreateRoom()
         }
         else
```

Comment:
I think you can just have an else if and else clause here instead of a nested if else clause. This would make the code neater.